doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -75,7 +75,7 @@ KRKNWK-19647: nRF53 has high power consumption in sleep mode
 
   **Affected platforms:** nRF5340
 
-.. rst-class:: v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0
+.. rst-class:: v3-0-0 v2-9-0-nRF54H20-1 v2-9-1 v2-9-0 v2-8-0
 
```

Comment:
@peknis Amazon Sidewalk has been moved to the add-on repository, and we'd like to apply the same solution we've done for Zigbee. Details regarding known issues and release notes for 3.0.0 will be listed in the add-on repository, and here we can remove the 3.0.0 tag, and just add the following note under the heading: 

```
After the |NCS| release v2.9.1, the Amazon Sidewalk protocol is no longer a part of the |NCS|.
For current known issues and release information, see the `Amazon Sidewalk documentation`_.
```